jobs/pull-kubernetes-e2e-gke-gci.sh

```diff
@@ -34,7 +34,16 @@ export JENKINS_USE_LOCAL_BINARIES=y
 export KUBE_FASTBUILD=true
 ./hack/jenkins/build.sh
 
-version=$(source build/util.sh && echo $(kube::release::semantic_version))
+# TODO(spxtr): once https://github.com/kubernetes/kubernetes/pull/35453 is in,
```

Comment:
We can't do that until we're no longer testing anything older than 1.5 (so a year or so).